desc/compute/_basis_vectors.py

```diff
@@ -0,0 +1,577 @@
+from desc.backend import jnp
+
+from .data_index import register_compute_fun
+from .utils import cross
+
+
+@register_compute_fun(
+    name="e_rho",
+    label="\\mathbf{e}_{\\rho}",
+    units="m",
+    units_long="meters",
+    description="Covariant radial basis vector",
+    dim=3,
+    params=[],
+    transforms={},
+    profiles=[],
+    data=["0", "R_r", "Z_r"],
+)
+def _e_sub_rho(params, transforms, profiles, data, **kwargs):
+    data["e_rho"] = jnp.array([data["R_r"], data["0"], data["Z_r"]]).T
+    return data
+
+
+@register_compute_fun(
+    name="e_theta",
+    label="\\mathbf{e}_{\\theta}",
+    units="m",
+    units_long="meters",
+    description="Covariant poloidal basis vector",
+    dim=3,
+    params=[],
+    transforms={},
+    profiles=[],
+    data=["0", "R_t", "Z_t"],
+)
+def _e_sub_theta(params, transforms, profiles, data, **kwargs):
+    data["e_theta"] = jnp.array([data["R_t"], data["0"], data["Z_t"]]).T
+    return data
+
+
+@register_compute_fun(
+    name="e_zeta",
+    label="\\mathbf{e}_{\\zeta}",
+    units="m",
+    units_long="meters",
+    description="Covariant toroidal basis vector",
+    dim=3,
+    params=[],
+    transforms={},
+    profiles=[],
+    data=["R", "R_z", "Z_z"],
+)
+def _e_sub_zeta(params, transforms, profiles, data, **kwargs):
+    data["e_zeta"] = jnp.array([data["R_z"], data["R"], data["Z_z"]]).T
+    return data
+
+
+@register_compute_fun(
+    name="e_rho_r",
+    label="\\partial_{\\rho} \\mathbf{e}_{\\rho}",
+    units="m",
```

Comment:
These are consistent with what they were before. Do we intentionally keep it as meters for simplicity?  If not the rho derivatives should introduce some unit cancellations.